### ISSUE TITLE
Exception raised when trying to sort a "Variant" sequence in Python 3

### ISSUE THREAD

**Issue by KelSolaar** [nerdvegas/rez#891]
While migrating some of our code to Python 3, I noticed that a few of our unit tests are breaking:

> TypeError("'<' not supported between instances of 'Variant' and 'Variant'")

So while we maybe should not have tried to sort `Variant`  sequences, it was certainly not raising an exception with Python 2.7, shouldn't it be possible to sort them by using the family and version of the underlying `FileSystemVariantResource` ?

Cheers,

Thomas

**Comment by nerdvegas** [comment]
Yeah it should absolutely be possible, I wonder what py2 is doing to allow
this. Do you mind ticketing it up?

Cheers
A

**Comment by KelSolaar** [comment]
> Do you mind ticketing it up?

As in creating a new feature ticket?

**Comment by nerdvegas** [comment]
Sounds more like a regression no? I'd just ticket it as a bug report,
variants should be sortable already imo.
A

**Comment by KelSolaar** [comment]
Oh right but I cannot label anything on the Rez repo :)

**Comment by nerdvegas** [comment]
Ignore me, this is already a ticket doh.

**Comment by KelSolaar** [comment]
Ah! You got me very confused!

**Comment by JeanChristopheMorinPerso** [comment]
I'm pretty lat to the poarty on this one @KelSolaar , but do you know if this was fixed since then? I know we did multiple Python 3 fixes since 2020.
